gadget-container/entrypoint/entrypoint.go

```diff
@@ -170,7 +171,9 @@ func installNRIHooks() error {
 		return fmt.Errorf("creating %s: %w", destinationPath, err)
 	}
 
-	err = copyFile(destinationPath, "/opt/hooks/nri/nrigadget", 0o640)
+	nriGadgetName := fmt.Sprintf("%s-nrigadget", os.Getenv("GADGET_NAMESPACE"))
+
+	err = copyFile(path.Join(destinationPath, nriGadgetName), "/opt/hooks/nri/nrigadget", 0o640)
```

Comment:
I guess this could be the same issue as the `gadgettracermanager`: What happens if we change something here from a version to another one. This would break possibly many things if there are different versions deployed in different namespaces.
But on the other hand `nrigadget` shouldn't change that often.